csharp/tests/Valkey.Glide.UnitTests/GildeStringTests.cs

```diff
@@ -7,6 +7,7 @@ public class GildeStringTests
     [Fact]
     public void Sorting()
     {
+        new GlideClient(new());
```

Comment:
Please revert, I think this caused a [panic in CI](https://github.com/valkey-io/valkey-glide/actions/runs/14599390669/job/40953509657?pr=3601#step:7:487):
```
thread '<unnamed>' panicked at library/core/src/panicking.rs:218:5:
unsafe precondition(s) violated: slice::from_raw_parts requires the pointer to be aligned and non-null, and the total size of the slice not to exceed `isize::MAX`
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread caused non-unwinding panic. aborting.
```

Reply:
Accidentially left this in (had to check whether the dll lookup still works), sry